lib/cocoapods/target.rb

```diff
@@ -140,14 +140,14 @@ def xcconfig_path(variant = nil)
     #         module map.
     #
     def umbrella_header_path
-      support_files_dir + "#{label}-umbrella.h"
+      support_files_dir + "#{product_module_name}.h"
```

Comment:
Won't this change only work if the umbrella header is the inferred one, but break if the module map has a custom umbrella header instead (like in `OHHTTPStubs` where `OHHTTPStubs.h` exists but **isn't** the umbrella header)?